includes/generate-sponsors.js

```diff
@@ -0,0 +1,169 @@
+/**
+ * @fileoverview
+ * This script generates a markdown file containing sponsor information.
+ * It fetches sponsor data from specified JSON files, formats
+ * the data into HTML sections, and writes the output to a new markdown file.
+ *
+ * Usage:
+ *   Run the script using Node.js:
+ *   node tools/generate-sponsors.js
+ *
+ * Author: Amaresh S M
+ */
+"use strict";
+
+//-----------------------------------------------------------------------------
+// Requirements
+//-----------------------------------------------------------------------------
+
+const fs = require("node:fs").promises;
+const path = require("node:path");
+const { stripIndents } = require("common-tags");
+
+//-----------------------------------------------------------------------------
+// Data
+//-----------------------------------------------------------------------------
+
+const SPONSORS_URL = path.resolve(__dirname, "../src/_data/sponsors.json");
+const TECH_SPONSORS_URL = path.resolve(
```

Comment:
Can we rename these to `SPONSORS_FILE_PATH` and `TECH_SPONSORS_FILE_PATH` as these are not URLs but file paths? Also, can we rename fetch* functions to read* as they're not fetching data from URLs but reading files?

Reply:
sure